### PR TITLE
Display last interaction result (if present)

### PR DESCRIPTION
**Pull request by timmyjose**
Story: https://www.pivotaltracker.com/n/projects/2184102/stories/164249035

### CODE REVIEW

**Comment by timmyjose** [comment]
> It looks like most records will display "Last interaction: no previous appointment". Maybe we should just remove that instead and display nothing? @dburka what do you think?

Yes, that's right. I have a version of the sandbox dump on my dev setup, and indeed most of the records show up as `No previous appointment`. There are quite a few `Visited` though and then a few `Agreed to visit` etc. In that order of frequency.

**Comment by timmyjose** [comment]
@dburka Hey Daniel, could you please comment on this PR? Basically, we want to figure out whether showing the last interaction status (along with the date) would be useful considering that the last interaction status for most patients would be `No previous appointment found`.

**Comment by timmyjose** [comment]
Closing this PR for now after discussion with @dburka. Story has been updated with some details of the discussion: https://www.pivotaltracker.com/n/projects/2184102/stories/164249035. 

I will create a new PR once the design discussion is finalised.

**Comment by timmyjose** [comment]
Reopening this since a new story (https://www.pivotaltracker.com/n/projects/2184102/stories/165908063) has been created for the changes suggested by @dburka.

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -36,4 +36,23 @@ def handle_impossible_registration_date(date, format: "%d-%b-%Y")
...
+
+    last_appointment = patient.appointments.order(scheduled_date: :desc).second
+    last_appointment_date = last_appointment.created_at.strftime("%d-%b-%Y")
+    interaction_result = "Last interaction: " + last_appointment_date
+
+    if last_appointment.agreed_to_visit.present?
+      interaction_result += ' - Agreed to visit'
+    elsif last_appointment.remind_on.present?
+      interaction_result += ' - Remind to call later'
+    elsif last_appointment.status == Appointment.statuses[:visited]
```
Comment (kitallis) [inline app/helpers/application_helper.rb]: `last_appointment.status_visited?`

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -36,4 +36,23 @@ def handle_impossible_registration_date(date, format: "%d-%b-%Y")
       date.strftime(format)
     end
   end
+
+  def show_last_interaction_date_and_result(patient)
+    return if patient.appointments.count < 2
+
+    last_appointment = patient.appointments.order(scheduled_date: :desc).second
```
Comment (kitallis) [inline app/helpers/application_helper.rb]: `last_interaction` + `last_interaction_date`?

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (timmyjose) [reply app/helpers/application_helper.rb]: Done.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (timmyjose) [reply app/helpers/application_helper.rb]: Done.

**Pull request merged**